homeassistant/components/html5/manifest.json

```diff
@@ -1,10 +1,12 @@
 {
   "domain": "html5",
   "name": "HTML5 Push Notifications",
-  "codeowners": [],
+  "codeowners": ["@alexyao2015"],
+  "config_flow": true,
   "dependencies": ["http"],
   "documentation": "https://www.home-assistant.io/integrations/html5",
   "iot_class": "cloud_push",
   "loggers": ["http_ece", "py_vapid", "pywebpush"],
-  "requirements": ["pywebpush==1.14.1"]
+  "requirements": ["pywebpush==1.14.1"],
+  "single_config_entry": true
```

Comment:
Why only single?

Reply:
Multiple html5 wouldn't make any sense. I don't know how that would be used and if it's even supported today. People have no way of selecting a particular vapid key in the frontend. If you have an idea of how multiple instances can be used, let me know and I would be happy to reenable this.